platform_it/src/test/java/com/splicemachine/test/SpliceTestPlatformConfig.java

```diff
@@ -294,6 +294,7 @@ public static Configuration create(String hbaseRootDirUri,
         config.set("splice.authentication.impersonation.users", "dgf=splice;splice=*");
         config.setBoolean("splice.authentication.impersonation.enabled", true);
         config.set("splice.authentication.ldap.mapGroupAttr", "jy=splice,dgf=splice");
+        config.setInt("splice.txn.completedTxns.cacheSize", 4096);
```

Comment:
How did you pick 4K?  The current default is 128K.  Would it be beneficial to cache more transactions?

Reply:
In real environment, the default remains 128K. This setting, 4096, is used only in standalone environment and, most importantly, integration tests. The goal was to fit ITs into 2GB for Splice 3.0 (HBase 2.0) and that cache was a large contributor to accumulating objects, which were not even needed in the context of rather short tests.

Reply:
Sure, I get that.  I was just asking why 4K was picked instead of 8K or 16K, etc.  The fix looks good.

Reply:
It's somewhat arbitrary: severely reduced to prevent object accumulation during ITs but still adequate for standalone workloads (including ITs). I wouldn't have a strong opinion against 8K or 2K.